### PR TITLE
fix: when click on a task title toggle the extra info

### LINKED ISSUE #1811: Clicking on done Task shouldn't take me to Goal
**Is your feature request related to a problem? Please describe.**
It's very annoying when behavior is inconsistent.

**Describe the solution you'd like**
When I click on a Task title it opens/closes the extra info (collapse).  
When I do the same thing on a done Task title it takes me to Goal - when it shoukd instead do the same as a todo Task.

**Describe alternatives you've considered**
None.

**Additional context**
None.

### PR DESCRIPTION
**Pull request by mehedihasan2810**
closes #1811 

Now clicking on a done task it will toggle the task extra info. other way to say it will behave like a accordion.

@tijlleenders

### CODE REVIEW

**Review by tijlleenders — DISMISSED** [review]
Thanks @mehedihasan2810 this looks good!

After code review by @Tushar-4781 we can merge this.  
It might take a week or so to get this through - be patient.

File: src/components/MyTimeComponents/MyTimeline/MyTimeline.tsx [hunk 1]
```diff
@@ -127,6 +127,10 @@ export const MyTimeline: React.FC<MyTimelineProps> = ({ day, myTasks, taskDetail
         //   setOpenReschedule({ ...task });
         // }
       } else if (actionName === "Done") {
+        const markDone = !!taskDetails[task.goalid]?.completedTodayIds.includes(task.taskid);
+
+        if (markDone) return;
+
```
Comment (Tushar-4781) [inline src/components/MyTimeComponents/MyTimeline/MyTimeline.tsx:133]: you can remove this line break.

**Review by Tushar-4781 — CHANGES_REQUESTED** [review]

File: src/components/MyTimeComponents/MyTimeline/MyTimeline.tsx [hunk 2]
```diff
@@ -180,33 +184,20 @@ export const MyTimeline: React.FC<MyTimelineProps> = ({ day, myTasks, taskDetail
               }
               onClick={() => {
                 if (displayOptionsIndex !== task.taskid) {
-                  if (markDone) {
-                    handleOpenGoal(task.goalid);
-                  } else {
-                    setDisplayOptionsIndex(task.taskid);
-                  }
-                } else setDisplayOptionsIndex("");
+                  setDisplayOptionsIndex(task.taskid);
```
[the remark this answers is not in the document]
Comment (mehedihasan2810) [reply src/components/MyTimeComponents/MyTimeline/MyTimeline.tsx:187]: ok

File: src/components/MyTimeComponents/MyTimeline/MyTimeline.tsx (hunk 1 above)
Comment (mehedihasan2810) [reply src/components/MyTimeComponents/MyTimeline/MyTimeline.tsx:133]: Hi @Tushar-4781 are you saying to remove the line break that is between these two lines of code or to remove these both lines?

**Comment by mehedihasan2810** [comment]
Hi @Tushar-4781, The changes are made

**Review by Tushar-4781 — APPROVED** [review]

**Pull request merged**